onnxruntime/core/providers/rocm/nn/conv_transpose.cc

```diff
@@ -93,9 +93,8 @@ Status ConvTranspose<T>::DoConvTranspose(OpKernelContext* context, bool dynamic_
       }
       s_.y_dims = gsl::make_span(y_dims);
 
-      if (w_dims_changed) {
+      if (w_dims_changed)
```

Comment:
We loosely follow google c++ style, so here is CUDA coding style issue, could you please sync it backward?

Reply:
You want me to update the CUDA EP source file conv_transpose.cc so that it follows the style?